pkg/operator/ceph/controller/controller_utils_test.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2020 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controller
+
+import (
+	"testing"
+
+	cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
+	"github.com/stretchr/testify/assert"
+)
+
+func CreateTestClusterFromStatusDetails(details map[string]cephv1.CephHealthMessage) cephv1.CephCluster {
+	return cephv1.CephCluster{
+		Status: cephv1.ClusterStatus{
+			CephStatus: &cephv1.CephStatus{
+				Details: details,
+			},
+		},
+	}
+}
+
+func TestIgnoreHealthErrForJustMdsAllDown(t *testing.T) {
```

Comment:
Typically we have a single function called `Test CanIgnoreHealthErrStatusInReconcile` and we test the various options inside it. Please merge into a single function. Thanks!

Reply:
Done